docs/get-started/getting-started-gcp.md

```diff
@@ -74,7 +74,10 @@ The script below will initialize with the following:
 ### Create a Dataproc Cluster using T4's
 * One 16-core master node and 5 32-core worker nodes
 * Two NVIDIA T4 for each worker node
-
+Note that for image-version 2.1, secure boot is enabled by default, please add disable the secure boot by adding `--no-shielded-secure-boot`
```

Comment:
This feels wrong to me. I don't think the default suggestion should be "disable secure boot". I think it would be better to point them at the google cloud docs on how to set up secure boot properly, and then provide the disable secure boot if and only if they are using an operating system version that is not supported.

Reply:
updated, thanks